doc/manual/manual.tex

```diff
@@ -1572,6 +1572,145 @@ \subsubsection{Free surface stabilization}
+  - (\mathbf u_s \cdot \mathbf g ) (1 - \phi) \kappa_s \rho_s
+  \notag
+  \\
+  &\quad
+  - K_D \mathbf g \cdot \nabla \rho_f .
+  \notag
+\end{align*}
+%
+The melt velocity is computed as
+\[
+ \mathbf u_f =  \mathbf u_s - \frac{K_D}{\phi} (\nabla p_f - \rho_f g),
+\]
+but is only used for postprocessing purposes and for computing the time step length.  
+
+\note{Here, we do not use the visco-elasto-plastic rheology of the \cite{KMK2013} formulation. 
+Hence, we do not consider the elastic deformation terms that would appear on the right hand side of Equation 
+\ref{eq:stokes-1-melt} and Equation \ref{eq:stokes-3-melt} and that include the elastic and compaction stress 
+evolution parameters $\xi_\tau$ and $\xi_p$. Moreover, our viscosity parameters $\eta$ and $\xi$ only cover viscous 
+deformation instead of combining visco-elasticity and plastic failure. This would require a modification of the rheologic 
+law using effective shear and compaction viscosities $\eta_\text{eff}$ and $\xi_\text{eff}$ combining a failure criterion 
+and shear and compaction visco-elasticities.}
+
+Moreover, melt transport requires an advection equation for the porosity field $\phi$:
+\begin{align}
+  \label{eq:porosity}
+  \rho_s \frac{\partial (1 - \phi)}{\partial t} + \nabla \cdot \left[ \rho_s (1 - \phi) \mathbf u_s \right]
+  &=
+  - \Gamma
+  & \quad
+  & \textrm{in $\Omega$},
+  i=1\ldots C
+\end{align}
+
+In order to solve this equation in the same way as the other advection equations, we replace the second term of the equation by: 
+
+\begin{equation*}
+\nabla \cdot \left[ \rho_s (1 - \phi) \mathbf u_s \right]
+= \left( 1-\phi \right) \left( \rho_s \nabla \cdot \mathbf u_s 
++ \nabla \rho_s \cdot \mathbf u_s \right)
+- \nabla \phi \cdot \rho_s \mathbf u_s 
+\end{equation*}
+Then we use the same method as for the mass conservation (described in section \ref{sec:boussinesq}) and assume the change in density is dominated by the change in static pressure, which can be written as
+$\nabla p_s \approx \nabla p_\text{static} \approx \rho_s \textbf{g}$.
+This finally allows us to write
+\begin{equation*}
+\frac{1}{\rho_s} \nabla \rho_s \cdot \mathbf{u_s} 
+\approx \frac{1}{\rho_s} \frac{\partial \rho_s}{\partial p_s} \nabla p \cdot \mathbf{u_s} 
+\approx \frac{1}{\rho_s} \frac{\partial \rho_s}{\partial p_s} \nabla p_s \cdot \mathbf{u_s} 
+\approx \frac{1}{\rho_s} \frac{\partial \rho_s}{\partial p_s} \rho_s \textbf{g} \cdot \mathbf{u_s} 
+\approx \kappa_s \rho_s \textbf{g} \cdot \mathbf{u_s} 
+\end{equation*}
+so we get
+\begin{equation*}
+\frac{\partial \phi}{\partial t} + \mathbf u_s \cdot \nabla \phi
+= \frac{\Gamma}{\rho_s}
++ (1 - \phi) (\nabla \cdot \mathbf u_s + \kappa_s \rho_s \textbf{g} \cdot \mathbf{u_s} ).
+\end{equation*}
+
+More details on the implementation can be found in \cite{DH2015}. A benchmark case demonstrating the propagation of solitary waves can be found in Section \ref{sec:benchmark-solitary_wave}.
```

Comment:
no-break space after `Section`.

I guess the reference is currently dead but will be filled in one the other PR will be merged?